docs/developer-docs/latest/update-migration-guides/migration-guides.md

```diff
@@ -3,29 +3,29 @@ title: Migrate Guides - Strapi Developer Documentation
 description: All the migration guides for a Strapi application.
 ---
 
-# Migrations guides
+# Migration guides
 
-## Instructions
+:::strapi Looking for the v3 to v4 migration guide?
+The v3 to v4 migration guides are located in the [v4 documentation](https://docs.strapi.io/developer-docs/latest/update-migration-guides/migration-guides.html#v3-to-v4-guides)
```

Comment:
```suggestion
The v3 to v4 migration guides are located in the [v4 documentation](https://docs.strapi.io/developer-docs/latest/update-migration-guides/migration-guides.html#v3-to-v4-migration-guide)
```

I've just double-checked [on the other PR](https://github.com/strapi/documentation/pull/740/files#diff-ae917440232d456eb0827dc0f96f9d36a76d824499d337d6d6c1ca6c014e0deaR32) and updated the link accordingly :-)